src/psyclone/psyir/transformations/psy_data_trans.py

```diff
@@ -58,23 +58,23 @@ class PSyDataTrans(RegionTrans):
     >>> from psyclone.parse.utils import ParseError
     >>> from psyclone.psyGen import PSyFactory
     >>> api = "gocean1.0"
-    >>> filename = "nemolite2d_alg.f90"
-    >>> ast, invokeInfo = parse(filename, api=api, invoke_name="invoke")
-    >>> psy = PSyFactory(api).create(invokeInfo)
+    >>> ast, invoke_info = parse(SOURCE_FILE, api=api)
+    >>> psy = PSyFactory(api).create(invoke_info)
     >>>
     >>> from psyclone.psyir.transformations import PSyDataTrans
     >>> data_trans = PSyDataTrans()
     >>>
     >>> schedule = psy.invokes.get('invoke_0').schedule
-    >>> schedule.view()
+    >>> if DEBUG:
```

Comment:
How about, rather than this `DEBUG` parameter, we simply comment-out the `schedule.view()` with a note to say 'uncomment to see text view of schedule', e.g.:

    >>> # Uncomment the following line to see a text view of the schedule
    >>> # schedule.view()

That's one less 'magic' parameter then.

Reply:
OK, all done.